src/SwedbankPay.Sdk/UserAgent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Reflection;
+
+namespace SwedbankPay.Sdk
+{
+    public static class UserAgent
+    {
+        public static string Default;
+        static UserAgent()
+        {
+            Default = $"swedbankpay-sdk-dotnet/{Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion}";
```

Comment:
I suspect `InformationalVersion` is going to be a bit too verbose. If GitVersion is responsible for providing its value, it's going to contain the Git SHA, for instance. I think `AssemblyFileVersionAttribute` is sufficient. I also think we should have a test that verifies that the value returned here contains a sensible version number and not just the default `1.0.0.0`.